### PR TITLE
Correct create_application en string

### PR DESCRIPTION
**Pull request by gileri**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | ?
| Documentation | no
| Translation   | yes
| Fixed tickets | no
| License       | MIT

Correct one typo in CoreBundle messages.en.yml.
I didn't manage to run tests yet.

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -359,7 +359,7 @@ quickstart:
     developer:
         title: 'Developers'
         description: 'We also thought to the developers: Docker, API, translations, etc.'
-        create_application: 'Create your third application'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: In fact we should be

> Create your third party application

**Comment by gileri** [comment]
Oh. I corrected this as i messages.fr.yml it's indicated "create your first application". Also this string is on a <a> which links to /developer, where there is a "How to create my first application" link.

I will amend this PR to correct as much languages I can.

**Comment by j0k3r** [comment]
I might be wrong, @wallabag/core ?

**Comment by gileri** [comment]
Ok, after review the files with Google Translate, here are the results.

da : not translated
de : "create an application and use the wallabag API"
en : "create your third application"
es : "create your third-party application"
fa : "create your application"
fr : "create your third-party application"
it : "create your application"
oc : "create your third-party application"
pl : "create your application"
pt : "create your third application"
ro : not translated
tr : not translated

@j0k3r I think you're right based on the most complete translations

**Comment by j0k3r** [comment]
Thanks for your first PR 🤘

**Comment by gileri** [comment]
Thanks to you for merging it !

**Pull request merged**